definitions/procedures/backup/clean.rb

```diff
@@ -0,0 +1,16 @@
+module Procedures::Backup
+  class Clean < ForemanMaintain::Procedure
```

Comment:
This it not being used right now anywhere.

Reply:
Clean up after failed backup was added